mlir/test/lib/Dialect/Linalg/TestLinalgRankReduceContractionOps.cpp

```diff
@@ -0,0 +1,68 @@
+//===- TestLinalgRankReduceContractionOps.cpp - Test Linalg rank reduce
+//contractions ---===//
```

Comment:
`Test Linalg rank reduce contractions` can be probably skipped to avoid wrapping to the new line.